public/adamant-api/asset.d.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
`adamant-api` files should not be in /public dir. Did you accidentally copy them while resolving the conflicts? @NikIvv

Reply:
However, the conflicts are still not resolved. Please fix it.